youtrack_sdk/client.py

```diff
@@ -341,3 +352,54 @@ def get_users(self, offset: int = 0, count: int = -1) -> Sequence[User]:
                 count=count,
             ),
         )
+
+    def get_issue_links(self, issue_id: str, offset: int = 0, count: int = -1) -> Sequence[IssueLink]:
+        """Read the list of links for the issue in YouTrack.
+
+        https://www.jetbrains.com/help/youtrack/devportal/resource-api-issues-issueID-links.html#get_all-IssueLink-method
+        """
+        return parse_obj_as(
+            tuple[IssueLink, ...],
+            self._get(
+                path=f"/issues/{issue_id}/links",
+                fields=model_to_field_names(IssueLink),
+                offset=offset,
+                count=count,
+            ),
+        )
+
+    def get_issue_link_types(self, offset: int = 0, count: int = -1) -> Sequence[IssueLinkType]:
+        """Read the list of all available link types in in YouTrack.
+
+        https://www.jetbrains.com/help/youtrack/devportal/resource-api-issueLinkTypes.html#get_all-IssueLinkType-method
+        """
+        return parse_obj_as(
+            tuple[IssueLinkType, ...],
+            self._get(
+                path="/issueLinkTypes",
+                fields=model_to_field_names(IssueLinkType),
+                offset=offset,
+                count=count,
+            ),
+        )
+
+    def link_issues(
+        self,
+        *,
+        source_issue_id: str,
+        target_issue_id: str,
```

Comment:
When I tested this, it was not obvious to me that I should use issue ID like `3-1230` instead of readable issue ID like `AB-123`. Whereas in `get_issue_links` I can use readable issue ID without any error. But I think it's not a code problem, it's a agreement problem. We should use IDs like `3-1230` everywhere to avoid such problems.

Reply:
You are right, funny how I got into same issue when implementing
So what I did is to add more info about the expected param value in the doc string